text/0010-burn-coretime-revenue.md

```diff
@@ -0,0 +1,31 @@
+# RFC-0010: Burn Coretime Revenue
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 19.07.2023                                                                   |
+| **Description** | Revenue from Coretime sales should be burned                                                                   |
+| **Authors**     |                   Jonas Gehrlein                                                                          |
+
+## Summary
+
+The Polkadot UC will generate revenue from the sale of available Coretime. The question then arises: how should we handle these revenues? Broadly, there are three potential paths – burning the revenue and thereby removing it from total issuance or divert it to the Treasury. This Request for Comment (RFC) presents arguments favoring burning as the preferred mechanism for handling revenues from Coretime sales.
+
+## Motivation
+
+How to handle the revenue accrued from Coretime sales is an important economic question that influences the value of DOT and should be properly discussed before deciding for either of the options. Now is the best time to start this discussion.
+
+## Stakeholders
+
+Polkadot DOT token holders.
+
+## Explanation
+
+This RFC discusses potential benefits of burning the revenue accrued from Coretime sales instead of diverting them to Treasury. Here are the following arguments for it.
+
+It's in the interest of the Polkadot community to have a consistent and predictable Treasury income, because volatility in the inflow can be damaging, especially in situations when it is insufficient. This raises the question: Why should we consider adding more volatility to the Treasury by tying it to fluctuating Coretime sales? Notably, Coretime revenue often appears to be inversely correlated with times when the Treasury should increase spending. During periods of low Coretime utilization (indicated by lower prices), it's important for the Treasury to spend more on projects and endeavours to increase the demand for Coretime. This pattern underscores that Coretime sales, by their very nature, are an inconsistent and unpredictable source of funding for the Treasury. Given the importance of maintaining a steady and predictable inflow, it's unnecessary to rely on another volatile mechanism. Some might argue that we could have both: a steady inflow (from inflation) and some added bonus from Coretime sales, but  burning the revenue would offer further benefits as described below.
```

Comment:
Would it be worth adding the proposed change in Treasury income from inflation to this same RFC so the entire change in tokenomics can be discussed in one place & ensures that reviewers have the complete picture?

Reply:
done in [a0a8345](https://github.com/polkadot-fellows/RFCs/pull/10/commits/a0a83453a7a87b2c1185f8952cfc18439ac5bde8). Nonetheless, my primary focus remains on the discourse surrounding the burn mechanism.